targets/tizenrt-artik05x/configs/jerryscript/Make.defs

```diff
@@ -1,5 +1,37 @@
+###########################################################################
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+###########################################################################
+#
+# Copyright 2017 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
```

Comment:
@glistening It seems that now the "JS Foundation and other contributors" copyright clause is gone and got replaced by company copyright notice. This should happen the other way around.

When the project was transferred to the Foundation, all "top-level" individual and company copyright notices were replaced by the catch-all "Copyright JS Foundation and other contributors" notice. (There are some files that have been imported into the project from external sources with non-Apache2 but compatible licenses. There, there appropriate notices are preserved below the Apache2 license block, see `jerry-libm/*.c` or `jerry-core/ecma/base/ecma-helpers-errol.c`. Thus the reference to "top-level" notices.) This has been documented in the Contribution Guidelines as well: https://github.com/jerryscript-project/jerryscript/blob/master/CONTRIBUTING.md#2-ensure-all-files-have-a-proper-license-header-and-copyright-notice 

> Any code that you want to contribute to the project must be licensed under the Apache License 2.0. Contributions under a different license can not be accepted.
> Adding copyright notices other than the project-wide notice ("Copyright JS Foundation and other contributors, http://js.foundation") is not permitted.